src/static/less/pages/for-filers.less

```diff
@@ -0,0 +1,161 @@
+// Mobile first!
+
+.for-filers {
+
+  .hero {
+    h1 {
+      padding-bottom: 5px;
+      background: transparent url() scroll no-repeat left center;
+      .avenir-next-bold;
+      .at2x( '../img/book-isocon.png', 42px, 42px );
```

Comment:
On the *for-filers* page, this Less mixin is generating a CSS rule that's looking for the non-existent `img/book-isocon_2x.png` image. If you have killer Photoshop skills you can try creating one using `img/icon_explore-data_2x.png` as a template. Otherwise, I can try digging up the original icons' Illustrator files. Let me know if that's needed.

![screen shot 2015-06-16 at 10 15 12 pm](https://cloud.githubusercontent.com/assets/1060248/8198504/65d8cb9e-1475-11e5-99e9-3584b97425c1.png)